x-pack/plugins/fleet/server/services/artifacts/errors.ts

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
We are exporting and defining all of our errors "./servers/errors" also all of our errors are inherting from `IngestManagerError` it's probably a good idea to keep things consistent here?